src/org/project/tests/Tests.java

```diff
@@ -31,5 +32,13 @@ private static void creatingAndCheckingCorrectDisplayOfAllProducts() {
         System.out.println(electronics);
         System.out.println(smartphone1);
         System.out.println(smartphone2);
+
+        //Testy ProductManager
```

Comment:
testy produkt managera powinny miec osobna metode, ktora bedzie miec osobna nazwe ktora to bedzie wskazywac na to co testujemy, tutaj przeczytaj sobie o teorii testow jednostkowych, zwlaszcza fragment o nazwach dla testow: https://modestprogrammer.pl/testy-jednostkowe-100-procent-tego-co-musisz-o-nich-wiedziec